### DISCUSSION TITLE
Github Campus expert for Beginner or not?

### DISCUSSION THREAD

**Discussion by arajshow** [community/community#37262] [Other Feature Feedback, Questions, & Ideas]
## Benefits of Github Campus Programm

In my college, many students applied for the GitHub campus expert program and now I am filling as I missed it so I  want to know the benefits and drawbacks of this post for any beginner developer.

**Comment by melvincwng** [comment]
@arajshow Hey Raj, you can refer to this link here https://education.github.com/experts to understand further the benefits of the GitHub Campus Experts program.

To summarize, this is a program whereby if selected, you will undergo training by GitHub to enhance your technical and non technical skills (e.g. public speaking, personal branding etc). 

These trainings essentially mould you into a better and stronger leader to help your own local community at school. You can even collaborate with other campus experts to host online or offline events such as hackathons, meetups, workshops etc.

Hope this clarifies and answers your question.
